src/main/java/com/napier/sem/CapitalCitiesDAO.java

```diff
@@ -0,0 +1,17 @@
+package com.napier.sem;
+
+import java.util.List;
+
+/**
+ * The interface Capital Cities dao.
+ */
+public interface CapitalCitiesDAO {
+    /**
+     * Gets all capital cities organized by largest population to smallest.
+     *
+     * @return all capital cities organized by largest population to smallest.
+     */
+    List<CapitalCities> getAllCapitalCities();
```

Comment:
By the javadocs comment it seems that a more appropriate name would be `getAllCapitalCitiesPopFromLargestToSmallest`